doc/tutorial/Tutorial.tex

```diff
@@ -664,10 +664,10 @@ \subsection{Existential quantification}
 Lemma refl_if : forall x : D, (exists y, R x y) -> R x x.
 \end{coq_example}
 
-Remark that the hypotheses which are local to the currently opened sections
+The hypotheses that are local to the currently opened sections
 are listed as local hypotheses to the current goals.
-The rationale is that these hypotheses are going to be discharged, as we
-shall see, when we shall close the corresponding sections.
+The rationale for this is that these hypotheses are going to be discharged, as
```

Comment:
Ok. I'm puzzled by this one. You set out, at least in part, to reduce word count. And now you are adding a pair of words to a perfectly idiomatic sentence (at least in technical writing). Any particular reason?